### PR TITLE
fix: Update `aten.embedding` to reflect schema

### PR DESCRIPTION
**Pull request by gs-olive**
# Description

- Remove arguments not present in initial schema for embedding
- Improve coverage of embedding operator by expanding set of convertible implementations
- Update parameter-checking function accordingly

**Note:** Reflects [this embedding schema](https://github.com/pytorch/pytorch/blob/3a300ed84efa8e458df02ee4f92532667cefbe29/aten/src/ATen/native/native_functions.yaml#L2194-L2199).

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ x ] My code follows the style guidelines of this project (You can use the linters)
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ x ] I have made corresponding changes to the documentation
- [ - ] I have added tests to verify my fix or my feature
  - Existing embedding tests
- [ x ] New and existing unit tests pass locally with my changes
- [ x ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py [hunk 1]
```diff
@@ -149,10 +135,9 @@ def aten_ops_embedding(
         name,
         input=args[1],
         weight=args[0],
-        max_norm=args_bounds_check(args, 2),
-        norm_type=args_bounds_check(args, 3),
-        scale_grad_by_freq=args_bounds_check(args, 4),
-        sparse=args_bounds_check(args, 5),
+        # args[2] is the padding index, which is useful for training only
+        scale_grad_by_freq=args_bounds_check(args, 3),
+        sparse=args_bounds_check(args, 4),
```
Comment (apbose) [inline py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:140]: Has the schema changed? I see below that` max_norm `and `norm_type` are not part of it?

Comment (gs-olive) [reply py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:140]: It seems so, as per this snippet from `native_functions.yaml`: https://github.com/pytorch/pytorch/blob/3a300ed84efa8e458df02ee4f92532667cefbe29/aten/src/ATen/native/native_functions.yaml#L2194-L2199

Comment (apbose) [reply py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:140]: The changes look good. Just that since `scale_grad_by_freq` and `sparse` are also not supported, should we not add a validator for them?
